### PR TITLE
Completed javascript-loops exercise. - Tiffany Brown

### PR DESCRIPTION
**Pull request by tiffanydbrown**
Screenshot 2023-10-25 at 2 12 04 PM

### CODE REVIEW

File: javascript-loops/main.js [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+}
+console.log('Value of doubleAll:', doubleAll([1, 2, 3, 4, 5])); // [2, 4, 6, 8, 10]
+console.log('Value of doubleAll:', doubleAll([2, 4, 6, 8, 10])); // [4, 8, 12, 16, 20]
+
+function getKeys(obj) {
+  const keys = [];
+
+  for (const key in obj) {
+    keys.push(obj[key]);
+  }
```
Comment (shawnkost) [inline javascript-loops/main.js:76]: You should only be pushing `key` in this one. This function is getting only the `property` or `key` names, whereas the other one actually gets the values that exist at that property

**Review by shawnkost — CHANGES_REQUESTED** [review]
One small change here Tiffany!

File: javascript-loops/main.js (hunk 1 above)
Comment (tiffanydbrown) [reply javascript-loops/main.js:76]: ok

Comment (tiffanydbrown) [reply javascript-loops/main.js:76]: I'm still getting the same thing. `for (const key in obj) {
    values.push(key);
  }`

**Comment by tiffanydbrown** [comment]
Screenshot 2023-10-25 at 2 29 43 PM

**Comment by tiffanydbrown** [comment]
Screenshot 2023-10-25 at 3 01 32 PM

Sorry, accidentally changed the wrong function.

**Review by shawnkost — APPROVED** [review]
👍🏻

**Pull request merged**